index.html

```diff
@@ -7,37 +7,192 @@
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+      
+
+      <section class="tonic-card">
+        <div class="container-image-4">
+        </div>
+          <section class="left-block">                
+              <div class="multi-post">
+                      <h2>Tonic</h2>
+                  <ul class="frame-2">
+                      <li class="canopy-text"><h3>CANOPY</h3></li>
+                      <li class="role-text"><h3>Back End Dev</h3></li>
+                      <li class="year-text"><h3>2015</h3></li>
+                  </ul>
+              </div>
+              <div class="text-paragraph">
+                  <p>
+                      A daily selection of privately personalized reads; 
+                      no accounts or sign-ups required.
+                  </p>
+              </div> 
+              
+              <ul class="primary-buttons">
+                  <li><button class="button1" type="button">html</button></li>
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+        
+        <section class="about-me-section">
+            <section>
+                <div class="about-myself">
+                    <h1>About Myself</h1>
+                </div>                
+                <p class="about-me-text">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="link-image">
+                    <p class="connect-link">LET’S CONNECT</p>
+                </div>
+                    <ul class="connect-icon">
+                        <li><a href="#"><img src="./image/Linkedinicon.png" alt="" id="image2" /></a></li>
+                        <li><a href="#"><img src="./image/github.png" alt="" /></a></li>
+                        <li><a href="#"><img src="./image/twitter.png" alt="" width="20" height="20" /></a></li>
+                        <li><a href="#"><img src="./image/M.png" alt="" width="20" height="20" /></a></li>
+                        <li><a href="#"><img src="./image/fingers.png" alt="" width="20" height="20" /></a></li>
+                    </ul>     
+                    <button class="get-resume" type="button">Get My Resume</button>   
+                    <div class="language">

```

Comment:
- [ ] I notice that some elements from your technologies list are missing Javascript technology and also your `arrow` , please add them and stick the design as much as possible (e.g., font, colors, images, text, margins ) :smile: . Check the screenshot and see the differences ; 

### Figma template
![Screenshot from 2022-07-08 00-04-25](https://user-images.githubusercontent.com/6615394/177891087-161e43fc-43bc-4ea2-a733-a86d9b442329.png)

### Your project
![Screenshot from 2022-07-08 00-04-01](https://user-images.githubusercontent.com/6615394/177891088-b8f9ff83-55bb-4580-8285-7ea4c1c515b1.png)